netbox/ipam/models/vlans.py

```diff
@@ -202,6 +202,21 @@ class VLAN(PrimaryModel):
         null=True,
         help_text=_("The primary function of this VLAN")
     )
+    qinq_svlan = models.ForeignKey(
+        to='self',
+        on_delete=models.PROTECT,
+        related_name='qinq_cvlans',
```

Comment:
What is cvlans? not sure if this was a typo, or customer - any harm in expanding it out? actually can it be quinq_service_vlan to be more friendly as to what it is instead of abbreviation?

Reply:
CVLAN (customer) and SVLAN (service) are pretty standard terms in the context of Q-in-Q. I think it's okay to leave them as-is, but we should be sensitive to any user feedback regarding terminology during the beta.